docs/pbc/all/accessibility-improvement/202410.0/base-shop/install-the-spryker-accessibility-improvements.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Accessibility improvements
+description: Learn how install accessibility improvements.
+last_updated: March 31, 2025
+template: feature-integration-guide-template
+---
+
+Keyboard accessibility improvements include the following changes:
+
+- Enhanced Accessibility: Made several interface elements more accessible. You can now navigate more logically using the Tab key, and screen readers provide better descriptions of elements.
+- Keyboard-Friendly Search: Improved the way keyboard users can interact with the product search suggestions.
+- Better Visual Clarity: Adjusted the color scheme across the site to ensure text stands out better from its background, making it easier for everyone to read.
+
+To install Yves keyboard accessibility improvements, take the following steps:
+
+1. Update necessary modules with the following command:
+```bash
+  composer update spryker-shop/shop-ui
+```
+
+2. Generate translation cache for Yves:
+- Add glossary keys for the new accessibility improvements.
```

Comment:
It's not about generation of translation cache. So suggest to put it into the separate section.